src/pages/group/group.ts

```diff
@@ -51,6 +52,27 @@ export class GroupPage {
 		this.arePlayersSelected = playersSelected.length > 0;
 	}
 
+	/**
+	 * Allows the point listed to become editabl.e
+	 */
+	public editPoints(): void {
+		this.isEditMode = !this.isEditMode;
+	}
+
+	/**
+	 * Allows the user to manually edit points. This is bound to the player.points (which
+	 * does not persist) in the UI. To be used when a mistake was made upon incorrect/accidental
+	 * use of updatePoints().
+	 */
+	public updatePlayerPoints(): void {
```

Comment:
Since this is basically the same as updatePoints we could merge the two into a single function. Then we could have a parameter, maybe just a boolean for now that triggers whether it performs the 'update points' operation from a successful vote or not.